backend/src/common/config.loader.js

```diff
@@ -31,6 +31,16 @@ export const config = {
     updateIntervalDelay: hoursToMs(Number(process.env.FEATURES_UPDATE_INTERVAL_DELAY)),
     globalIterationDelay: Number(process.env.RUNNER_GLOBAL_ITERATION_DELAY),
   },
+  externalApiUrls: {
```

Comment:
Good! I would separate more into services, host and apis:

```
externalServices: {
  steamWeb: {
    host: "https://store.steampowered.com",
    appPage: "app"
  },
  steamApiOfficial: {
    host: "https://api.steampowered.com",
    currentPlayersForApp: "ISteamUserStats/GetNumberOfCurrentPlayers/v1/?appid="
    appList: "ISteamApps/GetAppList/v2",
  },
  steamApiUnofficial: {
    host: "https://store.steampowered.com/api",
    appDetails: "appdetails?appids="
  },
}
```
Be aware when you use this, I did not put the "/" in the host at the end so in the steam client you will have to add them between the host and apis like this: 
```
const service = this.#options.externalServices.steamWeb;
const url = `${service.host}/${service.appPage}/${id}`;

```
.